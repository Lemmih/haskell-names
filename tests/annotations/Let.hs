module Let where

a =
  let x = 2
      y = 3
  in let
      t = x y z t
      z = 4
      y = 2
  in x y z
