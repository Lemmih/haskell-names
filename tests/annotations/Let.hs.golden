x        at 7:11 is a local value defined at 4:7
y        at 7:13 is a local value defined at 9:7
z        at 7:15 is a local value defined at 8:7
t        at 7:17 is a local value defined at 7:7
x        at 10:6 is a local value defined at 4:7
y        at 10:8 is a local value defined at 9:7
z        at 10:10 is a local value defined at 8:7
